cms/templates/widgets/sock.html

```diff
@@ -47,12 +47,14 @@ <h2 id="sock-heading" class="title sr-only">${_("{studio_name} Documentation").f
             'sr_mouseover_text': _('Send an email to {email}').format(email=partner_email),
             'text': _('Contact Us'),
             'condition': bool(partner_email)
-        }, {
+        }]
+
+        links.append({
```

Comment:
This isn't a change we can push upstream, as is.
How can we move this into a template/fragment?

Reply:
thus why we need to discuss...

Reply:
because the change could move these dicts into settings, python, separate mako file, etc...